src/c++/perf_analyzer/genai-perf/genai_perf/main.py

```diff
@@ -118,13 +118,13 @@ def create_graphs(stats: Statistics) -> None:
     plot_manager.create_default_graphs()
 
 
-def finalize():
+def finalize(args: Namespace):
```

Comment:
Beautiful PR, Matt! Thank you for the clear description.

What are your thoughts on only passing the profile_export_file for now rather than the whole Namespace? I'm okay with either approach. Just curious what your thoughts are here.

Reply:
Done

Reply:
Awesome, appreciate you getting this done so quickly!